### PR TITLE
Add galaxy training event

### PR DESCRIPTION
**Pull request by Marie59**
Adding a galaxy training event in May. @bgruening is it okay ?

### CODE REVIEW

File: content/events/2023-05-training-Fair-Ease/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Galaxy training: Fair-Ease feat EuroScienceGateway"
+date: '2023-05-16'
+days: 2
+tease: "[FAIR-EASE and EuroScienceGateway join forces](https://eosc.eu/news/stronger-together-fair-ease-and-eurosciencegateway-join-forces).
+continent: EU
+location:
+  name: In-person, Lille
+gtn: false
```
Comment (bgruening) [inline content/events/2023-05-training-Fair-Ease/index.md:9]: we use GTN isn't it?

File: content/events/2023-05-training-Fair-Ease/index.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+contact: "Marie Josse"
+subsites: [all]
+---
+
+First steps for the Fair-Ease partners on Galaxy with a training session on both the user and the developpement side of the platform.
```
Comment (bgruening) [inline content/events/2023-05-training-Fair-Ease/index.md:14]:
```suggestion
First steps for the EOSC FAIR-Ease partners on Galaxy with a training session on both the user and the tool developpement side of the platform.
```

File: content/events/2023-05-training-Fair-Ease/index.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+title: "Galaxy training: Fair-Ease feat EuroScienceGateway"
+date: '2023-05-16'
+days: 2
+tease: "[FAIR-EASE and EuroScienceGateway join forces](https://eosc.eu/news/stronger-together-fair-ease-and-eurosciencegateway-join-forces).
+continent: EU
+location:
+  name: In-person, Lille
+gtn: false
+contact: "Marie Josse"
+subsites: [all]
```
Comment (bgruening) [inline content/events/2023-05-training-Fair-Ease/index.md:11]:
```suggestion
tags: [esg, eosc]
subsites: [all]
```

File: content/events/2023-05-training-Fair-Ease/index.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Galaxy training: Fair-Ease feat EuroScienceGateway"
+date: '2023-05-16'
+days: 2
+tease: "[FAIR-EASE and EuroScienceGateway join forces](https://eosc.eu/news/stronger-together-fair-ease-and-eurosciencegateway-join-forces).
+continent: EU
+location:
+  name: In-person, Lille
```
Comment (bgruening) [inline content/events/2023-05-training-Fair-Ease/index.md:8]:
```suggestion
  name: Lille (France)
```

File: content/events/2023-05-training-Fair-Ease/index.md (hunk 1 above)
Comment (Marie59) [reply content/events/2023-05-training-Fair-Ease/index.md:9]: Right ;) I will put true

**Review by bgruening — APPROVED** [review]

**Pull request merged**